python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -237,12 +247,15 @@ def runDataRef(self, dataRef):
             photoCalib = calexp.getPhotoCalib()
 
         icSourceCat = dataRef.get("icSrc", immediate=True)
+        sfdSourceCat = dataRef.get("src", immediate=True)
```

Comment:
ProcessCcdWithFakesConnections needs to be updated with cT.Input() entries for sfdSourceCat so this can work in Gen3.